lib/util.js

```diff
@@ -1057,3 +1057,55 @@ exports._exceptionWithHostPort = function(err,
 // process.versions needs a custom function as some values are lazy-evaluated.
 process.versions[exports.inspect.custom] =
   (depth) => exports.format(JSON.parse(JSON.stringify(process.versions)));
+
+
+// Callbackify section
+const CBFY_ERR = 'The last argument to the callbackified function must be a ' +
+                 'callback style function';
+
+
+function _cbfy_onRejected(rej, cb) {
+  // !rej guard inspired by BlueBird https://goo.gl/t5IS6M
+  if (!rej) {
+    const newRej = new Error(rej + '');
+    newRej.cause = rej;
+    rej = newRej;
+  }
+  return cb(rej);
```

Comment:
It looks like you removed the `process.nextTick` call, so this behavior will be incorrect if the callback function throws an error.